modules/exploits/linux/http/ispconfig_lang_edit_php_code_injection.rb

```diff
@@ -0,0 +1,237 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
```

Comment:
I alluded to this elsewhere but a more reliable way to clean up the payload would be to define a `#cleanup` method and exploit the vulnerability again to call PHP's `unlink` method instead of `file_put_contents`. That would ensure that if a session is not established, that the file is still removed. This of course assumes the vulnerability can be exploited multiple times to execute arbitrary PHP code. Per the "Reliability" note, this should be the case.